Week3/hw/app1.js

```diff
@@ -0,0 +1,6 @@
+// 1. Create a function that takes 3 arguments and returns the sum of the these arguments.
+
+function f(a, b, c) {
```

Comment:
This looks nice! But could you change the name of the function `f` is not really descriptive:)